### ISSUE TITLE
Can't install Puphpeteer

### ISSUE THREAD

**Issue by louis-genestier** [nesk/puphpeteer#77]
I tried to install Puphpeteer using `composer require nesk/puphpeteer` but I get an error:  
```bash
Using version ^1.6 for nesk/puphpeteer
./composer.json has been updated
Loading composer repositories with package information
Updating dependencies (including require-dev)
Restricting packages listed in "symfony/symfony" to "5.0.*"
Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - nesk/rialto 1.3.0 requires symfony/process ^3.3|^4.0 -> no matching package found.
    - nesk/rialto 1.2.1 requires symfony/process ^3.3|^4.0 -> no matching package found.
    - nesk/rialto 1.2.0 requires symfony/process ^3.3|^4.0 -> no matching package found.
    - nesk/puphpeteer 1.6.0 requires nesk/rialto ^1.2.0 -> satisfiable by nesk/rialto[1.2.0, 1.2.1, 1.3.0].
    - Installation request for nesk/puphpeteer ^1.6 -> satisfiable by nesk/puphpeteer[1.6.0].

Potential causes:
 - A typo in the package name
 - The package is not available in a stable-enough version according to your minimum-stability setting
   see <https://getcomposer.org/doc/04-schema.md#minimum-stability> for more details.
 - It's a private package and you forgot to add a custom repository to find it

Read <https://getcomposer.org/doc/articles/troubleshooting.md> for further common problems.

Installation failed, reverting ./composer.json to its original content.
```

**Comment by katzsimon** [comment]
Any updates on getting this to work with symfony/process 5?

**Comment by nesk** [comment]
Fixed in nesk/rialto v1.4.0
